bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -550,12 +536,12 @@ public String getRecommendedMinExp() {
         public static final Mask DEFAULT = MASK_IMAGING;
 
         // The internal values of height and width for this instance of Mask
-        private double _width = Size.IMAGING_SIZE;
-        private double _height = Size.IMAGING_SIZE;
-        private String _displayValue;
-        private String _logValue;
+        private final double _width;
```

Comment:
Since the default values here for `_width` and `_height` were rightfully changed from `Size.IMAGING_SIZE` to not being assigned until the constructor (which requires values for these parameters so one would assume they were always created assigned to whatever was specified) to allow the members to be marked as `final`, I am guessing that the defaults were never actually used in any way?